### ISSUE TITLE
add parser for subscriptions.json/likes.csv files

### ISSUE THREAD

**Issue by purarue** [seanbreckenridge/google_takeout_parser#2]

**Comment by karlicoss** [comment]
pre-2019: seems that subscriptions were in `YouTube/subscriptions/subscriptions.opml`
but actually it's not an opml file at all, looks like HTML for the youtube subscription page as if it was downloaded without logging into the account
so it basically just has the google login form o_o

**Comment by karlicoss** [comment]
for me, liked videos are
- pre-2020: `playlists/likes.json` (seems like it should be handled by `_parse_json_activity`)
- after that: `Liked videos.csv` 

also perhaps makes sense to parse other playlists?

**Comment by seanbreckenridge** [comment]
yeah, can parse other playlists

one other concern is how to merge this across takeouts, not sure if playlists would return individual items or the entire playlist as one object/event

not sure how to track which playlist is newer/if playlist items should be overwritten or not (probably not?)

**Comment by karlicoss** [comment]
yeah, I guess for playlists makes sense to return the latest only
not sure what happens to deleted videos, might be interesting to check if they disappear from playlists
